compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__completion__completions_for_record_labels.snap

```diff
@@ -0,0 +1,148 @@
+---
+source: compiler-core/src/language_server/tests/completion.rs
+expression: "completion(TestProject::for_source(code), Position::new(6, 23))"
```

Comment:
I can't tell what is happening in these snapshots. It's a lot of data and the expression doesn't show what is being tested at all. Could we make them readable please 🙏

Reply:
I'm not entirely sure how to make this more readable honestly. I'll add a comment in the code itself saying something like " // completion inside parens below includes labels" so that at least it is a bit more clear what the point of the test is

Reply:
oh actually lemme see if I can make the expression at least show the actual code

Reply:
lmk if this helps or if there was something in particular you had in mind